apis/keda/v1alpha1/scaledobject_webhook.go

```diff
@@ -193,7 +193,7 @@ func verifyFallback(incomingSo *ScaledObject, action string, _ bool) error {
 		scaledobjectlog.WithValues("name", incomingSo.Name).Error(err, "validation error")
 		metricscollector.RecordScaledObjectValidatingErrors(incomingSo.Namespace, action, "incorrect-fallback")
 	}
-	return nil
+	return err
```

Comment:
> this would likely introduce undesired regression on denying SOs with fallback and multiple triggers with at least one being cpu or memory - [#5962 (review)](https://github.com/kedacore/keda/pull/5962#pullrequestreview-2416584056)

so, the code deny  trigger which has cpu or memory in function **CheckFallbackValid**  is wrong? so here, it's better to return nil and not take effect?

Reply:
I see the official website(https://keda.sh/docs/2.16/reference/scaledobject-spec/#fallback) and the code (scaledobject_types.go function CheckFallbackValid), they all  forbidden  CPU / memory fallback.  I think the initial purpose is to forbidden the metric type which is not AverageValue, and don't know why  forbidden  CPU / memory fallback? I see that CPU / memory also support type **AverageValue**. So, if I think there should be a conclusion whether forbidden it. If not forbidden, I think just change  code to allow the  CPU / memory, but webhook have to take effect to support the value validation. If forbidden, so just make it take effect.

Besides, I also make other code contribute, 
(https://github.com/kedacore/keda/pull/6344)
(https://github.com/kedacore/keda/pull/6350), and hope you to review, thank you.

Reply:
there is still value in printing the error in this case and maybe we should consider enhancing the user experience by throwing an event too. CPU / memory metrics are not owned by `external.metrics.k8s.io` API but instead by `metrics.k8s.io` API. That means KEDA can't fabricate the metric in a way that HPA would scale to fallback numbers.